NuKeeper.Abstractions/RegexMatch.cs

```diff
@@ -0,0 +1,24 @@
+using System.Text.RegularExpressions;
+
+namespace NuKeeper.Abstractions
+{
+    public static class RegexMatch
+    {
+        public static bool IncludeExclude(string target, Regex include, Regex exclude)
+        {
+            return
+                IsIncluded(target, include)
+                && !IsExcluded(target, exclude);
+        }
+
+        private static bool IsIncluded(string target, Regex regex)
+        {
+            return regex == null || regex.IsMatch(target);
+        }
+
+        private static bool IsExcluded(string target, Regex regex)
+        {
+            return regex != null && regex.IsMatch(target);
+        }
+    }
```

Comment:
If you want to go expression crazy:
```csharp
public static bool IncludeExclude(string target, Regex include, Regex exclude)
    => IsIncluded(target, include) && !IsExcluded(target, exclude);
private static bool IsIncluded(string target, Regex regex) => regex?.IsMatch(target) ?? true;
private static bool IsExcluded(string target, Regex regex) => regex?.IsMatch(target) ?? false;
```

Reply:
Hm, I'll think on that. I'm not crazy about complex ternary operator or complex one-liner lambda expressions.
But this might be under the line.